test/compile_test.ts

```diff
@@ -2,6 +2,7 @@ import {compilePackBundle} from '../testing/compile';
 import {executeFormulaOrSyncWithVM} from '../testing/execution';
 import {newMockSyncExecutionContext} from '../testing/mocks';
 import path from 'path';
+import { readFileSync } from 'fs';
```

Comment:
Nit: fix formatting to match other imports (i.e., no spaces around the imported symbol name).

Reply:
Done. Also added a eslint rule and use `lint-fix` to fix other instances.